lib/file.js

```diff
@@ -205,7 +205,13 @@ internals.openStat = function (path, mode, callback) {
     Fs.open(path, mode, function(err, fd) {
 
         if (err) {
-            return callback(Boom.notFound());
+            if (err.code === 'ENOENT') {
+                return callback(Boom.notFound());
+            } else if (err.code === 'EACCES' | err.code === 'EPERM') {
```

Comment:
Ok, I fixed it with additional mocking. I suspect a more proper solution would be to write a new module that converts `syserr` to `Boom` errors and use that instead. Any thoughts on such a module @arb ?